examples/step-20/doc/intro.dox

```diff
@@ -129,7 +134,62 @@ discontinuous elements of class $DQ(k)$ for the pressure $p$. For details
 about these spaces, we refer in particular to the book on mixed finite element
 methods by Brezzi and Fortin, but many other books on the theory of finite
 elements, for example the classic book by Brenner and Scott, also state the
-relevant results.
+relevant results. In any case, with appropriate choices of function
+spaces, the discrete formulation reads as follows: Find ${\mathbf
+u}_h,p_h$ so that
+@f{eqnarray*}
+  A(\{{\mathbf u}_h,p_h\},\{{\mathbf v}_h,q_h\}) = F(\{{\mathbf v}_h,q_h\})
+  \qquad\qquad \forall {\mathbf v}_h,q_h.
+@f}
+
+
+Before continuing, let us briefly pause and show that the choice of
+function spaces above provides us with the desired local conservation
+property. In particular, because the pressure space consists of
+discontinuous piecewise polynomials, we can choose the test function
+$q$ as the function that is equal to one on any given cell $K$ and
+zero everywhere else. If we also choose ${\mathbf v}=0$ (remember that
```

Comment:
Since you emphasized that you chose $q$ equal to one on a single cell $K$, maybe you should also emphasize that you choose ${\mathbf v}=0$ on *every* cell.

Reply:
Wolfgang, I think you mean $v=1$ here.

Reply:
No, you really need to choose v=0 everywhere.

Reply:
ah yes, sorry. I should have looked at the weak form.